.github/scripts/clang-format-check-all.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+THIS_PATH="$(realpath "$0")"
+THIS_DIR="$(dirname "$THIS_PATH")"/../CEF_App/src/
+FILE_LIST="$(find "$THIS_DIR" | grep -E ".*(\.cpp|\.hpp)$")"
+
+clang-format --dry-run -i --style=file --Werror $FILE_LIST
```

Comment:
This will fail if someone checks out the repo to a path that contains spaces :(

A couple of options - either brave the bash arrays:
```bash
readarray -d '' FILE_LIST < <(find "$THIS_DIR" \( -name '*.cpp' -o -name '*.hpp' \) -print0)

clang-format -i --style=file --Werror "${FILE_LIST[@]}"
```
or globstar:
```bash
shopt -s globstar # enables **
shopt -s nullglob # a non-matching glob expands to nothing rather than erroring
clang-format -i --style=file --Werror "$THIS_DIR"/**/*.cpp "$THIS_DIR"/**/*.hpp
```

Reply:
The globstar option seems easier to understand when reading the script - converted to globstar